babel.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = (api) => {
         "@babel/preset-env",
         {
           targets: {
-            node: "12.13.0",
+            node: "14.16.0",
```

Comment:
It should be `14.15.0` (https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md#14.15.0), i.e. first LTS, let's fix it for all PRs and ping me when it will be ready, thanks